README.md

```diff
@@ -2,7 +2,11 @@
 
 [![Maven Central](https://img.shields.io/maven-central/v/com.cloudant/kafka-connect-cloudant.svg)](http://search.maven.org/#search|ga|1|g:"com.cloudant"%20AND%20a:"kafka-connect-cloudant")
 
-This project includes Apache Kafka Connect source & sink connectors for IBM Cloudant.
+This project includes [Apache Kafka](https://kafka.apache.org/) [Connect](https://kafka.apache.org/documentation.html#connect) source and sink connectors for IBM Cloudant.
+
+These connectors can stream events:
+- **from** Cloudant (source connector) to Kafka topic(s)
+- **to** Cloudant (sink connector) from Kafka topic(s)
```

Comment:
I think maybe worth adding a note that it is also compatible with Apache CouchDB.
```suggestion
- **to** Cloudant (sink connector) from Kafka topic(s)

_Note:_ the connectors are also compatible with Apache CouchDB.
```